docs/main/Solutioning/openebs-on-kubernetes-platforms/rwx.md

```diff
@@ -0,0 +1,496 @@
+---
+id: rwx
+title: Provisioning Read-Write-Many (RWX) PVCs
+keywords: 
+  - Read-Write-Many
+  - RWX
+  - NFS server
+  - Provisioning Read-Write-Many (RWX) PVCs
+description: In this document, you learn about Provisioning Read-Write-Many (RWX) PVCs.
+---
+
+# Provisioning Read-Write-Many (RWX) PVCs
+
+Provisioning Read-Write-Many (RWX) Persistent Volume Claims (PVCs) involves setting up storage that can be simultaneously accessed in read-write mode by multiple nodes or pods within a Kubernetes cluster.
+
+Block storage, such as Persistent Volumes (PVs) in Kubernetes, is generally not designed to be mounted to multiple pods simultaneously due to several technical limitations and potential issues:
+
+- **File System Consistency**
+
+  Block storage devices are designed for exclusive access. Mounting the same block device to multiple pods can lead to file system corruption due to concurrent writes and uncoordinated access. Traditional file systems like ext4 or XFS are not designed to handle multiple writers, leading to inconsistencies and corruption.
+
+- **Read/Write Conflicts**
+
+  When multiple pods attempt to read from and write to the same block storage simultaneously, it can cause conflicts and data corruption. Block storage lacks the necessary coordination mechanisms to manage concurrent access, which is why shared access is problematic.
+
+- **Lack of Coordinated Locking**
+
+  File systems on block storage rely on locking mechanisms to ensure data integrity during concurrent access. These locks are not shared across different instances of the file system, leading to the risk of data corruption when the same block storage is mounted to multiple pods.
+
+- **Performance Issues**
+
+  Concurrent access by multiple pods can lead to performance degradation. The storage system might not be optimized for handling simultaneous I/O operations from multiple sources, causing bottlenecks and reduced performance.
```

Comment:
The above 4 sections tend to downplay the importance of block storage which I think we shouldn't be doing. We just need to highlight the use-cases of rwx, not try to downgrade block.

Reply:
Agreed ! sure, we can change this.